notebook/static/notebook/less/celltoolbar.less

```diff
@@ -12,6 +12,10 @@
     padding-right: 4px;
     .hbox();
     .end();
+    // safari fix, we cannot use -webkit-flex on hbox
+    // and vbox either or all css get borked
+    // cf https://github.com/jupyter/nbgrader/issues/394
+    display: -webkit-flex;
```

Comment:
Technically that what we should do, but as I stated in the
comment it is broken on safari if in hbox or vbox. 
Reposted screenshot on linked issue:

![](https://cloud.githubusercontent.com/assets/335567/9584924/3e3deeea-5014-11e5-807b-5fb3e838cbb3.png)